Cargo.toml

```diff
@@ -14,10 +14,11 @@ too_many_arguments = 'allow'
 anyhow = { version = "1.0.95", default-features = false }
 heck = { version = "0.5", default-features = false }
 js-component-bindgen = { version = "1.11.0" }
-orca-wasm = { version = "0.9.2", default-features = false }
+wirm = { version = "2.1.0", features = ["parallel"] }
```

Comment:
This shouldn't use the `"parallel"` feature, and probably use `default-features = false` again (though I don't know what the default features are, and hence whether we could use them.)

Reply:
Correct, changed it back.